index.html

```diff
@@ -16,13 +16,16 @@
             <div class="logo">
              <a href="index.html"><h2 class="hero">Mloman</h2></a>
             </div>
-            <ion-icon class="ion-icon hambuger" name="menu-outline" id="menu"></ion-icon>
+            <div class="hambuger-menu">
+                <i class="fa-solid fa-bars" id="menu"></i>
+                <!-- <i class="fa-solid fa-xmark hambuger" id="close"></i> -->
+            </div>
             <ul>
-                <li><a href="">Home</a></li>
-                <li><a href="">About</a></li>
-                <li><a href="">Portfolio</a></li>  
-                <li><a href="">Contact</a></li>
-                <li><a href=""><ion-icon class="ion-icon mail" name="mail-outline"></ion-icon></a></li>
+                <li><a href="#">Home</a><i class="fa-solid fa-greater-than"></i></li>
+                <li><a href="#about">About</a><i class="fa-solid fa-greater-than"></i></li>
+                <li><a href="#portfolio">Portfolio</a><i class="fa-solid fa-greater-than"></i></li>  
+                <li><a href="#contact">Contact</a><i class="fa-solid fa-greater-than"></i></li>
+                <li><a href="#"><ion-icon class="ion-icon mail" name="mail-outline"></ion-icon></a></li>
```

Comment:
- [ ] Good job working on this and on the animation too, but it does not look exactly like the design. Kindly make it full height of the page as the design and include the background image.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/167607131-4f8adf98-1516-46de-8513-b5df7cb765eb.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/167607029-22234a16-397c-44cc-b548-10a91666603b.png)

Reply:
- [ ] As suggested by the previous reviewer kindly make the mobile menu of the full height of the page as the design and include the background image as per the Figma design.
- [ ] Optional : It is highly recommended to follow the same colour for (  > ) on your mobile menu. Also, follow the font size for all menu options.

https://user-images.githubusercontent.com/70269507/167625520-9648aea5-bde8-42df-968d-dbd50fd821bc.mov